benchmarks/util/nlte.py

```diff
@@ -0,0 +1,72 @@
+from collections import OrderedDict
+from copy import deepcopy
+from pathlib import Path
+
+from benchmarks.util.base import Base
+from tardis.io.atom_data import AtomData
+from tardis.io.configuration.config_reader import Configuration
+from tardis.io.util import yaml_load_file, YAMLLoader
+from tardis.model import SimulationState
+
+
+class Nlte:
```

Comment:
Please capitalize this as NLTE. I realize that is not Pythonic, but it is physics!

Reply:
Sure, I got you. The change is in commit: 67a73c978b316ffa22e04ec805c4a4eb441d6b29.